### PR TITLE
[TerritorialCouncil][CoPol] Add referent messages page

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch12319]

File: src/Entity/TerritorialCouncil/PoliticalCommitteeFeedItem.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use App\Entity\AbstractFeedItem;
+use App\Entity\Adherent;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\TerritorialCouncil\PoliticalCommitteeFeedItemRepository")
+ *
+ * @Algolia\Index(autoIndex=false)
```
Comment (ottaviano) [inline src/Entity/TerritorialCouncil/PoliticalCommitteeFeedItem.php:13]: no need anymore )

File: src/Repository/TerritorialCouncil/PoliticalCommitteeFeedItemRepository.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            ->addSelect('adherent')
+            ->leftJoin('feedItem.author', 'adherent')
+            ->where('feedItem.politicalCommittee = :politicalCommittee')
+            ->orderBy('feedItem.createdAt', 'DESC')
+            ->setParameter('politicalCommittee', $politicalCommittee)
+            ->setFirstResult($firstResultIndex)
+            ->setMaxResults($limit)
+        ;
+
+        return new Paginator($qb);
```
Comment (ottaviano) [inline src/Repository/TerritorialCouncil/PoliticalCommitteeFeedItemRepository.php:37]: you can use PaginatorTrait too

File: src/Repository/TerritorialCouncil/TerritorialCouncilFeedItemRepository.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            ->addSelect('adherent')
+            ->leftJoin('feedItem.author', 'adherent')
+            ->where('feedItem.territorialCouncil = :territorialCouncil')
+            ->orderBy('feedItem.createdAt', 'DESC')
+            ->setParameter('territorialCouncil', $territorialCouncil)
+            ->setFirstResult($firstResultIndex)
+            ->setMaxResults($limit)
+        ;
+
+        return new Paginator($qb);
```
Comment (ottaviano) [inline src/Repository/TerritorialCouncil/TerritorialCouncilFeedItemRepository.php:36]: same here

**Review by ottaviano — APPROVED** [review]

**Pull request merged**